server/room-battle.ts

```diff
@@ -491,6 +492,7 @@ export class RoomBattle extends RoomGames.RoomGame {
 	turn: number;
 	rqid: number;
 	requestCount: number;
+	dataResolvers?: DataResolver[];
```

Comment:
```suggestion
	dataResolvers?: ((args: string[]) => void)[];
```

I think it's more readable to put the type definition directly here, instead of forcing people to look up the definition for `DataResolver`. Especially since the definition is only used once, and is reasonably concise.